src/planets/Planet.ts

```diff
@@ -75,9 +84,39 @@ export default abstract class Planet extends AstronomicalObject implements IPlan
         return spherical2rectangular(coords);
     }
 
-    public abstract getHeliocentricEclipticSphericalJ2000Coordinates(): Promise<EclipticSphericalCoordinates>;
+    public async getHeliocentricEclipticSphericalJ2000Coordinates(): Promise<EclipticSphericalCoordinates> {
+        const cacheKey = `${this.name}_heliocentric_spherical_j2000`;
+
+        return this.getHeliocentricEclipticSphericalCoordinates(cacheKey, 'J2000');
+    }
+
+    public async getHeliocentricEclipticSphericalDateCoordinates(): Promise<EclipticSphericalCoordinates> {
+        const cacheKey = `${this.name}_heliocentric_spherical_date`;
+        const resourceSuffix = this.useVsop87Short ? 'DateShort' : 'Date';
+
+        return this.getHeliocentricEclipticSphericalCoordinates(cacheKey, resourceSuffix);
+    }
+
+    private async getHeliocentricEclipticSphericalCoordinates(
+        cacheKey: string,
+        resourceSuffix: string
+    ): Promise<EclipticSphericalCoordinates> {
+        return await getAsyncCachedCalculation(cacheKey, this.t, async () => {
+            const vsop87 = await import(this.getVsop87ImportResourceName(resourceSuffix)) as Vsop87;
```

Comment:
There is a problem with dynamic imports when using the bundled files.
I used dynamic imports for the besselian elements and wanted to use the bundle in nextjs. But it failed. The required besselian elements were not build (`next build`) into my final app.

But I will investigate this later